pkg/landscaper/blueprints/store.go

```diff
@@ -120,7 +120,7 @@ func NewStore(log logr.Logger, baseFs vfs.FileSystem, config config.BlueprintSto
 		gcConfig: config.GarbageCollectionConfiguration,
 	}
 
-	if config.Size != "0" {
+	if config.Size != "0" && config.Size != "" {
```

Comment:
hm the size should be also correctly defaulted as @MrBatschner mentioned

Reply:
I have merged the default config and the provided config. The other initialisations from my first commit are then no longer necessary.